### PR TITLE
Buildtools replacement

### PR DESCRIPTION
**Pull request by HON95**
### Status

- [X] Done
- [ ] Work in progress
- [ ] Needs testing from others

### Description
#201. Remove Vagrant, Make, etc. and update Docker Compose setup. Replace Drone CI with Travis CI, and self-hosted image repo with Docker Hub. Includes config and script samples for deployment.

### Checklist

- [ ] The code is linted (Dunno, didn't touch it)
- [X] My changes requires change to the documentation.
- [X] I have updated the documentation.
- [X] I have introduced changes to build configuration.

### Minor Changes

- Add configuration samples
- Add convenience scripts
- Applied some hardening to the Django app configuration

### Breaking Changes

- Basically replaced the entire build system

### Other
- [Live demo](https://studlan.haavard.tech)
- [Travis CI page](https://travis-ci.com/HON95/studlan)
- [Docker Hub page](https://hub.docker.com/r/hon95/studlan)

### CODE REVIEW

**Comment by MathiasFl** [comment]
Looks good, mucho good work
